pkg/apis/autoscaling/v1/fleetautoscaler.go

```diff
@@ -158,11 +158,11 @@ type CounterPolicy struct {
 	// Key is the name of the Counter. Required field.
 	Key string `json:"key"`
 
-	// MaxCapacity is the maximum aggregate Counter total capacity across the fleet.
+	// MaxCapacity is the maximum aggregate Counter available capacity across the fleet.
```

Comment:
Total for all the ones for fleet autoscaler.
```suggestion
	// MaxCapacity is the maximum aggregate Counter total capacity across the fleet.
```

Reply:
Changed!